deployments/docker/cluster-distributed-deployment/inventory.ini

```diff
@@ -0,0 +1,50 @@
+[dockernodes]
+dockernode01
+dockernode02
+dockernode03
+
+[admin]
+ansible-controller
+
+[coords]
+dockernode01
+
+[nodes]
+dockernode02
+
+[dependencies]
+dockernode03
+
+[docker:children]
+dockernodes
+coords
+nodes
+dependencies
+
+[docker:vars]
+ansible_python_interpreter= /usr/bin/python3
+StrictHostKeyChecking= no
+
+; Setup variables to controll what type of network to use when creating containers.
+dependencies_network= host
+nodes_network= host
+
+; Setup varibale to controll what version of Milvus image to use.
+image= milvusdb/milvus-dev:master-20220412-4781db8a
+
+; Setup static IP addresses of the docker hosts as variable for container environment variable config.
+etcd_ip= 10.170.0.19
```

Comment:
Why hardcode the ip?

Reply:
It's better to add a step to fetch the node ip, and register a variable.

Reply:
Register variable will have impact due to host or group name change. Example: if I reference from Hostvar[etcd], it will work ok as I've desitined etcd to run on a host named "etcd", but that case might not be true. If people have a different host name, then they would have to either change the host at every reference point or running into problem.

Reply:
But how can you make sure the ip address are accessible in different hosts.

Reply:
When people download this playbook, the first thing they need to do in order to get it running is to modify the inventory file, change the host name and IP address so it will be pass to container env variable.